server/src/main/java/org/apache/iotdb/db/service/DataNode.java

```diff
@@ -101,6 +109,11 @@ private DataNode() {
   private static final RegisterManager registerManager = new RegisterManager();
   public static ServiceProvider serviceProvider;
 
+  /** store the list when registering in config node for preparing trigger related resources */
+  private List<TriggerInformation> triggerInformationList;
```

Comment:
Maybe it's better to maintain the `triggerInformationList` (informations about trigger) in an independent class? Because in the future CQ and other features are integrated, the variables amount of `DataNode` maybe large.

Reply:
Got it.